internal/view/pod.go

```diff
@@ -487,20 +487,39 @@ func podIsRunning(f dao.Factory, path string) bool {
 	return re.Phase(po) == render.Running
 }
 
+var (
```

Comment:
Thanks for the quick update Jayson! I don't think we need to intro a sentinel err for this.
From the call site point of view, we infer the os or not? 
Does this make sense?